include/triton/core/tritonserver.h

```diff
@@ -89,7 +91,7 @@ struct TRITONSERVER_ServerOptions;
 ///   }
 ///
 #define TRITONSERVER_API_VERSION_MAJOR 1
-#define TRITONSERVER_API_VERSION_MINOR 10
+#define TRITONSERVER_API_VERSION_MINOR 11
```

Comment:
Please verify if this requires a version bump or not

Reply:
Yes. Any change in this API should come with a version bump.